packages/yoroi-extension/app/ergo-connector/stores/ConnectorStore.js

```diff
@@ -582,6 +580,51 @@ export default class ConnectorStore extends Store<StoresMap, ActionsMap> {
     }
     return result ?? [];
   }
+  static createAuthEntry: ({|
```

Comment:
Maybe alternatively consider putting these `static` methods in a new file `app/ergo-connector/api/index.js` ?